doc/source/io.rst

```diff
@@ -28,6 +28,11 @@ a web URL, for example for GeoJSON files from `geojson.xyz <http://geojson.xyz/>
     url = "http://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_110m_land.geojson"
     df = geopandas.read_file(url)
 
+You can also load ZIP files that contains ESRI shapefiles::
```

Comment:
I think it can contain anything, not just shapefiles. Am I wrong? I actually never tested it.

Reply:
@martinfleis yes, any of OGR's vector formats. The shapefile case is where it's most useful, because a shapefile is always 3 or more files, but GeoJSON might be zipped to save space.